### PR TITLE
Update default timeline behavior to only show tweets since last cache modified time with optional -f/--full flag

### PR DESCRIPTION
**Pull request by prologic**

### CODE REVIEW

File: cache.go [hunk 1]
```diff
@@ -45,6 +45,14 @@ func (cache Cache) Store(configpath string) {
 	}
 }
 
+func CacheLastModified(configpath string) (time.Time, error) {
+	stat, err := os.Stat(fmt.Sprintf("%s/cache", configpath))
```
Comment (quite) [inline cache.go:49]: Stat:ing feels a bit iffy to me. But adding some last updated timestamp to the cache file would imply changing its format, too... hmm

**Review by quite — CHANGES_REQUESTED** [review]
I would prefer it this was an option, not the default. Avoiding to change the current behaviour.

I guess stat:ing the file is a good enough solution for now, instead of changing the cache file format...

**Comment by prologic** [comment]
> I would prefer it this was an option, not the default. Avoiding to change the current behaviour.

How about a user-configurable  option  to change the default? e.g:

```#!yaml
TimelineView: nwest
```

> I guess stat:ing the file is a good enough solution for now, instead of changing the cache file format...

Yes I'm avoding Cache incompatibility here. Stat'ing the file is fine IHMO because most file systems support this. I'm using this  technique elsewhere as well in [rss2twtxt](https://github.com/prologic/rss2twtxt)

File: cache.go (hunk 1 above)
Comment (prologic) [reply cache.go:49]: As stated above I _think_ this is an acceptable approach IHMO.

**Comment by quite** [comment]
> How about a user-configurable option to change the default? e.g:
> 
> ```
> TimelineView: nwest
> ```

Yes, that's good! Don't forget to add to `config.yaml.example`

**Comment by quite** [comment]
@prologic i reworked this a bit. simplified slightly

**Pull request closed without merging**